docs/components/DataList/DataList.stories.mdx

```diff
@@ -246,6 +246,112 @@ children and inserts it above the list of items.
 </DataList>
 ```
 
+#### Clearing all filters
+
+##### In all filter groups (e.g. with the "Clear Filters" button)
+
+You can add a button that will clear all the filters by using the combination of
+the `Combobox` and `Button` components (check out the `Code` tab in the "Clear
```

Comment:
I think it may be worth adding a link to the Clear All Filters story with something like
`[Clear All Filters Story](../?path=/story/components-lists-and-tables-datalist-web--clear-all-filters)

Reply:
Great suggestion!